ember-table/src/components/row/expandable-row/index.ts

```diff
@@ -1,7 +1,123 @@
-import TableRow from '../table-row';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+//@ts-nocheck
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import { SelectionMode } from '../../../constants/table';
+import { TableMeta } from '../../table';
+
+import type { RowApi } from '@gavant/glint-template-types/types/ember-table/row';
+
+export interface TableExpandedRowArgs {
+    api: RowApi;
+    tableMeta?: TableMeta<{ expandedRowComponent: typeof Component; expandedRows?: any[] }>;
+    rowSelectionMode: SelectionMode;
+    onClick?(...args: any[]): void;
+    onDoubleClick?(...args: any[]): void;
+}
+
+interface TableExpandedRowSignature {
+    Args: TableExpandedRowArgs;
+    Blocks: {
+        default: [
+            {
+                api: RowApi;
+                cellValue: any;
+                cellMeta: any;
+                columnValue: any;
```

Comment:
There a reason this isn't some of these types don't extend the types we have for them?